services/compute/virtualmachine/virtualmachine.go

```diff
@@ -565,7 +565,8 @@ func (c *client) getVirtualMachineStorageProfileOsDisk(d *wssdcloudcompute.Disk)
 		return &compute.OSDisk{}
 	}
 	return &compute.OSDisk{
-		Vhd: &compute.VirtualHardDisk{URI: &d.Diskname},
+		Name: &d.Diskname,
+		Vhd:  &compute.VirtualHardDisk{URI: &d.Diskname},
```

Comment:
remove VHD URI

Reply:
I think we need to confirm that this won't affect any teams. This could be fixed in another PR and do a test run on it to make sure removing it is not breaking anything.

Reply:
Sounds good